### DISCUSSION TITLE
Hello from Germany

### DISCUSSION THREAD

**Discussion by SirRanjid** [OpenMined/courses#161] [Introduce Yourself]
Hello! Sven here. I'm just a nerd interested in computer science collecting knowledge along the way and hoping to put it to good use.
I just wanted to thank everyone involved for their hard work on making and running this project. So thank you - I really appreciate it!

**Comment by yemikifouly** [comment]
Hello Sven! Welcome to OpenMined and the discussion board! 🙂 Thank you so much! In the name of the whole team, you are very welcome! We are very glad you are enjoying the course. 😀
